multicluster/config/overlays/leader-ns/coverage/manager_command_patch_coverage.yaml

```diff
@@ -8,6 +8,6 @@ spec:
     spec:
       containers:
         - command: ["/bin/sh"]
-          args: ["-c", "/antrea-mc-controller-coverage -test.run=TestBincoverRunMain -test.coverprofile=antrea-mc-controller.cov.out leader --config=/controller_manager_config.yaml; while true; do sleep 5 & wait $!; done"]
+          args: ["-c", "/antrea-mc-controller-coverage leader --config=/controller_manager_config.yaml; while true; do sleep 5 & wait $!; done"]
```

Comment:
looks like these multicluster coverage manifests were not updated to reflect the introduction of the `run-cov-binary.sh` wrapper script?